hrui/phpunit.xml.dist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<phpunit backupGlobals="false" backupStaticAttributes="false" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" processIsolation="false" stopOnFailure="false" syntaxCheck="false" bootstrap="tests/phpunit/bootstrap.php">
+  <testsuites>
+    <testsuite name="hrui Extension Test Suite">
```

Comment:
For consistency with the other phpunit.xml files, can you please change this to HRUI Extension Test Suite? I don't think this extension has any name other than HRUI, right?